src/Expectation.lua

```diff
@@ -76,7 +76,9 @@ function Expectation.new(value)
 	local self = {
 		value = value,
 		successCondition = true,
-		condition = false
+		condition = false,
+		matchers = {},
```

Comment:
Is `matchers` used? I don't see it anywhere else.

Reply:
We use it once when we need to negate our expectation
https://github.com/Roblox/testez/pull/142/files#diff-c51d01b32afee27efe59f26103c1e704R129

We pass in the non-bound versions of the extend config.

Reply:
Oh, yep. There it is.